app/controllers/users/profiles_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Users::ProfilesController < ApplicationController
+  before_action :authenticate_user!
+
+  # プロフィール新規作成画面
+  def new
+    @profile = ProfileForm.new
+  end
+
+  # プロフィール作成
+  def create
+    @profile = ProfileForm.new(profile_form_params)
+    if @profile.save
+      redirect_to users_basic_path(current_user.id)
+    else
+      render :new
+    end
+  end
+
+  # プロフィール編集画面
+  def edit
+    @profile = ProfileForm.new
+  end
+
+  # プロフィール編集
+  def update
+    @profile = ProfileForm.new
+    # 複数のattributeをまとめて更新
+    @profile.assign_attributes(params, current_user)
```

Comment:
記事の通りに書いて、引数変えた方がいいですね
```suggestion
    @profile.assign_attributes(params)
```